src/components/subscription-details/components/Edit/PlanSelector.tsx

```diff
@@ -0,0 +1,113 @@
+import { h, FunctionalComponent } from '@stencil/core';
+import FixedFeature from 'components/shared/FixedFeature';
+import MeteredFeature from 'components/shared/MeteredFeature';
+import ConfigurableFeature from 'components/shared/ConfigurableFeature';
+import PlanCard from 'components/shared/PlanCard';
+import {
+  PlanFixedFeatureEdge,
+  PlanMeteredFeatureEdge,
+  PlanConfigurableFeatureEdge,
+} from '../../../../types/graphql';
+import store from '../../data/store';
+import { selectPlan, getSelectedPlan, setState } from '../../data/actions';
+
+const PlanMenu: FunctionalComponent = () => {
+  const { plans, selectedPlanId } = store.state.edit;
+  if (!plans) {
+    return null;
+  }
+
+  return (
+    <ul class="ManifoldSubscriptionCreate__PlanSelector__Menu">
+      {plans.map(({ node: plan }) => (
+        <li>
+          <label>
+            <input
+              type="radio"
+              value={plan.id}
+              checked={plan.id === selectedPlanId}
+              onClick={() => {
+                selectPlan(plan.id);
+              }}
+            />
+            <PlanCard plan={plan} isChecked={plan.id === selectedPlanId} />
+          </label>
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+const PlanDetails: FunctionalComponent = () => {
+  const plan = getSelectedPlan();
+
+  if (!plan) {
+    return null;
+  }
+
+  const { configuredFeatures, isLoading } = store.state.edit;
+
+  return (
+    <div
+      class="ManifoldSubscriptionCreate__PlanSelector__Details"
+      itemscope
+      itemtype="https://schema.org/IndividualProduct"
```

Comment:
I love that you’re using schema.org types! They’re like unfurl metadata—they really enhance perceived quality and benefit for customers. It’s a fancy touch that people appreciate.

But that said… I only realized now that these components only display for logged-in dashboards. Its meant to really help for public-facing pages. For anything that’s not search engine bot-reachable, though, it can be skipped. I don’t think it has any a11y benefits.

Reply:
this was just inherited all the way through from the React plan selector :P I didn't even know about schema.org until I started working here haha